courses/free-courses-pl.md

```diff
@@ -108,6 +108,9 @@
 * [Kurs Python 3](https://www.youtube.com/playlist?list=PLdBHMlEKo8UcOaykMssI1_X6ui0tzTNoH)
 * [Python 3 - Kurs wideo](https://www.youtube.com/playlist?list=PLbOPmSDkHx2pCboufcEKkinpUuramshmr)
 * [Raspberry Pi kurs od podstaw](https://forbot.pl/blog/kurs-raspberry-pi-od-podstaw-wstep-spis-tresci-id23139) - Piotr Bugalski (FORBOT.pl)
+* [Free Python course by freecodecamp](https://youtu.be/eWRfhZUzrAc)
```

Comment:
Please order the courses alphabetically, and also please give the same name as the video name and add author

Reply:
Also looks like these courses are English, so please move them to the corresponding file

Reply:
Also looks like these courses are in English. Please move them to the corresponding file